examples/pygi_mpl_numpy/README.rst

```diff
@@ -66,3 +66,19 @@ The include section requires the Python packages numpy, matplotlib (which are ta
         dateutil
         pyparsing
 
+Compiling
```

Comment:
I'd call this section something like 'glib schemas'  - it's not compiling in the sense we usually use the term (compiling C code).